contracts/scripts/v0.1/admin-aggregator/generate-aggregator-deployments.cjs

```diff
@@ -1,29 +1,16 @@
 const { getFormattedDate, loadJson, storeJson } = require('../utils.cjs')
 const { ethers } = require('hardhat')
 
-const priceFeeds = [
-  'BNB-USDT',
-  'BTC-USDT',
-  'BUSD-USDT',
-  'DAI-USDT',
-  'DOT-USDT',
-  'ETH-USDT',
-  'KLAY-USDT',
-  'MATIC-USDT',
-  'SOL-USDT',
-  'TRX-USDT',
-  'USDC-USDT',
-  'MNR-KRW'
-]
+const priceFeeds = ['CHF-USD', 'EUR-USD', 'GBP-USD', 'JPY-USD', 'KRW-USD']
```

Comment:
I think we should not modify this file if we use it. In the mid-term, we should pass names of price feeds as parameters. Please make an issue for it 🙏

Reply:
yes I have made an issue, thank you

https://github.com/Bisonai/orakl/issues/925